max-statements-per-line misses braceless if-else statement

**What version of ESLint are you using?**
2.8.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
```json
{
    "rules": {
        "max-statements-per-line": ["error", { "max": 1 }]
    }
}
```


**What did you do? Please include the actual source code causing the issue.**

When running eslint on the following file:

```js
var a = 1;

if (a) a *= 1; else b *= 1;

if (a) { a *= 1; } else { b *= 1; }
```

**What did you expect to happen?**

I would have expected both of these if statements to be flagged as having two many statements.

**What actually happened? Please include the actual, raw output from ESLint.**

I get this:

```
/private/tmp/blah/temp.js
  5:10  error  This line has too many statements. Maximum allowed is 1  max-statements-per-line
```


Is this intentional, or an oversight? If the latter, I can probably look at sending in a PR.

--- a/lib/rules/max-statements-per-line.js
+++ b/lib/rules/max-statements-per-line.js
@@ -99,6 +99,18 @@ module.exports = {
             enforceMaxStatementsPerLine(node.consequent);
         }
 
+        /**
+         * Check each line in both sides of an if statement
+         * @param {ASTNode} node node to evaluate
+         * @returns {void}
+         * @private
+         */
+        function checkLinesInNonBlockElse(node) {
+            if (node.alternate && node.alternate.type !== "BlockStatement") {
+                enforceMaxStatementsPerLine([node.alternate]);
+            }
+        }
+
         //--------------------------------------------------------------------------
         // Public API
         //--------------------------------------------------------------------------
@@ -106,7 +118,8 @@ module.exports = {
         return {
             Program: checkLinesInBody,
             BlockStatement: checkLinesInBody,
-            SwitchCase: checkLinesInConsequent
+            SwitchCase: checkLinesInConsequent,
+            IfStatement: checkLinesInNonBlockElse
         };
 
     }

--- a/tests/lib/rules/max-statements-per-line.js
+++ b/tests/lib/rules/max-statements-per-line.js
@@ -74,6 +74,7 @@ ruleTester.run("max-statements-per-line", rule, {
         { code: "var bar = 1; var baz = 2;", errors: [{ message: "This line has too many statements. Maximum allowed is 1." }] },
         { code: "var bar = 1; var baz = 2;", options: [{ max: 1 }], errors: [{ message: "This line has too many statements. Maximum allowed is 1." }] },
         { code: "if (condition) var bar = 1; if (condition) var baz = 2;", options: [{ max: 1 }], errors: [{ message: "This line has too many statements. Maximum allowed is 1." }] },
+        { code: "if (condition) var bar = 1; else var baz = 1;", options: [{ max: 1 }], errors: [{ message: "This line has too many statements. Maximum allowed is 1." }] },
         { code: "if (condition) { } if (condition) { }", options: [{ max: 1 }], errors: [{ message: "This line has too many statements. Maximum allowed is 1." }] },
         { code: "if (condition) { var bar = 1; } else { }", options: [{ max: 1 }], errors: [{ message: "This line has too many statements. Maximum allowed is 1." }] },
         { code: "if (condition) { } else { var bar = 1; }", options: [{ max: 1 }], errors: [{ message: "This line has too many statements. Maximum allowed is 1." }] },

